content-security-policy/support/testharness-helper.sub.js

```diff
@@ -41,6 +41,13 @@ function generateUrlWithPolicies(host, policy) {
     return url + "?policy=" + encodeURIComponent(policy);
 }
 
+function generateUrlWithAllowCSPFrom(host, allowCspFrom) {
+  var url = generateURL(host, PolicyHeader.ALLOW_CSP_FROM);
+  if (!allowCspFrom)
+    return url + "?allow_csp_from=";
```

Comment:
I'm pretty sure that this is still going to treat `""` and `null` the same (because both are `== false`).